cmd/acikubectl/cmd/debug.go

```diff
@@ -368,7 +372,21 @@ func clusterReport(cmd *cobra.Command, args []string) {
 				args: nodeItem.argFunc(systemNamespace, podName,
 					nodeItem.cont, nodeItem.args),
 			})
+
+			//Prepare kubectl cp command for aci-conatiners-host
+			tempName = fmt.Sprintf("hostfiles/node-%s/aci-containers-host", node.Name)
```

Comment:
Instead of "aci-containers-host", I think we should ideally put it in "aci-containers/k8s-pod-network" to maintain consistency with the directory structure on the host.

Reply:
Resolviing